src/EmptyCriteria/WhenNull.php

```diff
@@ -4,8 +4,29 @@
 
 namespace Yiisoft\Validator\EmptyCriteria;
 
+/**
+ * Empty criteria is a callable identifying when and which values exactly must be considered as empty for according
+ * rules to be skipped or not skipped at all.
+ *
+ * With this criteria, a rule is skipped only when the validated value is `null`.
+ *
+ * Can be set:
+ *
+ * - At a rule level via `$skipOnEmpty` property, but only for rules implementing {@see SkipOnEmptyTrait} / including
+ * {@see SkipOnEmptyTrait}.
+ * - At validator level ({@see Validator::$defaultSkipOnEmptyCriteria}).
+ *
+ * There is no shortcut for this criteria, because it's considered less used. Use new instance directly:
+ * `new WhenNull()`.
```

Comment:
```suggestion
 * Empty criteria is a callable returning true if a value must be considered empty.
 *
 * With `WhenNull` a rule is considered empty only when the value is `null`.
 *
 * Can be used:
 *
 * - At a rule level via `$skipOnEmpty` property, but only for rules implementing {@see SkipOnEmptyTrait} / including
 * {@see SkipOnEmptyTrait}.
 * - At validator level ({@see Validator::$defaultSkipOnEmptyCriteria}).
 *
 * There is no shortcut for this criteria, because it's considered less used. Use new instance directly:
 * `new WhenNull()`.
```